dcos_installer/async_server.py

```diff
@@ -360,6 +361,10 @@ def start(cli_options):
     make_default_config_if_needed('genconf/config.yaml')
     loop = asyncio.get_event_loop()
     app = build_app(loop)
+
+    if options.verbose:
```

Comment:
Instead of verbose, should it be a debug? 

(that's my only comment. everything else LGTM).